components/body/desktop.styl

```diff
@@ -62,6 +62,7 @@
     position relative
     top 50%
     transform translateY(80%)
+    -moz-transform translateY(-50%)
```

Comment:
It appears different in the firefox browser. I tried to vertically center the content using the display table method along with a few others and targeting the specific browser was the only way that I could get it center correctly.